.evergreen/functions.yml

```diff
@@ -331,7 +331,6 @@ functions:
         wait: true
     # copy ssh key (this key will be used to connect to ssh server)
     - command: shell.exec
-      silent: true
```

Comment:
Drive-by: `silent` would need to be set on the `params:`, not the command, if we wanted it to have an effect – but this should be fine either way, so let's just drop the invalid extra key